packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -106,15 +115,16 @@ describe(`request-queue`, function() {
       expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
       expect(callbacks.requestWillEnqueue.calledWith(sinon.match.has('request')))
           .to.be.true;
-
+      expect(performance.now() - startTime).to.be.above(800);
```

Comment:
Can you please make use of sinon clock to very reliably add do what ever timeout and clock stuff you want.

http://sinonjs.org/releases/v3.2.1/fake-timers/